src/app/app.component.spec.ts

```diff
@@ -34,14 +19,6 @@ describe('AppComponent', () => {
     expect(app).toBeTruthy();
   });
 
-  it('should initialize the app', async () => {
-    TestBed.createComponent(AppComponent);
```

Comment:
I think we could keep this line since it tests that the App Component was created